doc/source/ray-overview/use-cases.rst

```diff
@@ -522,3 +522,39 @@ The following are highlighted examples utilizing Ray AIR to implement end-to-end
         :type: ref
         :text: [Gallery] Full Ray AIR Examples Gallery
         :classes: btn-link btn-block stretched-link AIRExamples
+
+Large Scale Workload Orchestration
+----------------------------------
+
+The following highlights feature companies leveraging Ray Core's distributed APIs to simplify the orchestration of large scale workloads.
+
+.. panels::
+    :container: container pb-4
+    :column: col-md-4 px-2 py-2
+    :img-top-cls: pt-5 w-75 d-block mx-auto
+
+    ---
+    :img-top: /images/ray_logo.png
+
+    +++
+    .. link-button:: https://www.businessinsider.com/openai-chatgpt-trained-on-anyscale-ray-generative-lifelike-ai-models-2022-12
```

Comment:
If you add https://archive.ph/UF8Z8 here it might it more likely for people to read this -- the other one is behind a pay wall. Or maybe there is a better publicly available article by a reputable source?

Reply:
Do you know if the link is stable? I replaced it with https://archive.ph/o/aNvFN/https://www.businessinsider.com/openai-chatgpt-trained-on-anyscale-ray-generative-lifelike-ai-models-2022-12

Reply:
I don't know which links are stable or not unfortunately -- https://archive.ph/UF8Z8 has been working since Dec 16 but who knows :)

Reply:
I guess your link is better, if it stops working at least it has the true one embedded in it